src/dto/account-attributes-stream.params.dto.ts

```diff
@@ -0,0 +1,33 @@
+import { IsOptional } from 'class-validator';
+import { Transform } from 'class-transformer';
+import { ApiProperty } from '@nestjs/swagger';
+import { propertyMetadata } from '@us-epa-camd/easey-common/constants';
+import { AllowanceProgram } from '@us-epa-camd/easey-common/enums';
+import { ErrorMessages } from '@us-epa-camd/easey-common/constants';
+
+import { AllowanceParamsDTO } from './allowance.params.dto';
+import { IsAllowanceProgram } from '../pipes/is-allowance-program.pipe';
+
+export class AccountAttributesStreamParamsDTO extends AllowanceParamsDTO {
```

Comment:
Rather than having a seperate streaming params dto you can remove page and perPage from the accountAttributes param dto and then extend that to create a PaginatedAccountAttributesParamsDTO that adds on page and perPage. This way you don't have to have all the repeated code. You can use the accountAttributesParamsDTO for streaming endpoint and the paginatedAccountAttributesParamsDTO for the other endpoint. Also page and perPage right now are not showing up as required params in swagger. Change ApiPropertyOptional() to ApiProperty()